commands/operator-sdk/cmd/build.go

```diff
@@ -183,13 +186,15 @@ func buildFunc(cmd *cobra.Command, args []string) {
 		buildTestCmd.Stderr = os.Stderr
 		err = buildTestCmd.Run()
 		if err != nil {
-			log.Fatalf("failed to build test binary: %v", err)
+			log.Fatalf("failed to build test binary: (%v)", err)
 		}
 		// if a user is using an older sdk repo as their library, make sure they have required build files
 		testDockerfile := filepath.Join(scaffold.BuildTestDir, scaffold.DockerfileFile)
 		_, err = os.Stat(testDockerfile)
 		if err != nil && os.IsNotExist(err) {
 
+			log.Info("Generating operator test files.")
```

Comment:
This could be a little more descriptive. It's not really test files. It's the build manifests needed for the test-framework.
```Go
log.Info("Generating build manifests for test-framework")
```

Same with the log.Fatal error if this fails.